IOSExercises/Classes/EXRPerson.h

```diff
@@ -0,0 +1,33 @@
+//
+//  EXRPerson.h
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/26/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "EXRColor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXRPerson : NSObject
+
+@property (strong, readwrite, nonatomic) NSString *image;
+
+@property (strong, readwrite, nonatomic) NSString *name;
+
+@property (strong, readwrite, nonatomic) NSDate *birthday;
+
+@property (strong, readwrite, nonatomic) EXRColor *color;
+
++ (instancetype)personNamed:(NSString *)aName
```

Comment:
Not an actual comment but a note: sometimes it's difficult to think up sentence-style methods. In case you find yourself lost, can stick to the usual `<object>With<Attribute>: <attribute>: <attribute>:`... eg:
```
personWithName:(NSString *)name
      birthday:(NSDate *)birthday
 imageFilepath:(NSString *)imageFilepath
         color:(EXRColor *)color
```